Jantimizer/ExperimentSuite/Helpers/TimerHelper.cs

```diff
@@ -20,13 +20,21 @@ public static Stopwatch GetWatchAndStart()
         public static TestTimeReport StopAndGetReportFromWatch(this Stopwatch watch, string expName, string dbName, string testName, string name)
         {
             watch.Stop();
-            return new TestTimeReport(dbName, testName, expName, name, watch.ElapsedMilliseconds);
+            return new TestTimeReport(dbName, testName, expName, name, GetNs(watch), watch.ElapsedMilliseconds);
         }
 
         public static TestCaseTimeReport StopAndGetCaseReportFromWatch(this Stopwatch watch, string expName, string dbName, string testName, string caseName, string name)
         {
             watch.Stop();
-            return new TestCaseTimeReport(dbName, testName, expName, caseName, name, watch.ElapsedMilliseconds);
+            return new TestCaseTimeReport(dbName, testName, expName, caseName, name, GetNs(watch), watch.ElapsedMilliseconds);
+        }
+
+        // https://stackoverflow.com/questions/2329079/how-do-you-convert-stopwatch-ticks-to-nanoseconds-milliseconds-and-seconds
+        private static decimal GetNs(Stopwatch watch)
+        {
+            decimal ticks = (decimal)watch.ElapsedTicks;
+            decimal nanoseconds = (ticks / Stopwatch.Frequency) * 1000000000;
```

Comment:
![image](https://user-images.githubusercontent.com/53914190/166137602-fb7e5af7-16d5-4b09-9978-dc6b2b66c63f.png)
Is it necssary for it to be a decimal? It seems to just create a bunch of unneeded 0's.